client/asset/eth/multirpc.go

```diff
@@ -1548,6 +1548,9 @@ const (
 	providerAlchemy     = "alchemy.com"
 	providerAnkr        = "ankr.com"
 	providerdRPC        = "drpc.org"
+	providerBlastAPI    = "blastapi.io"
+	providerLlamaRPC    = "llamarpc.com"
+	providerPublicNode  = "publicnode.com"
```

Comment:
Should remove all the ones we know are not working now. so remove "drpc.org", "ankr.com", "flashbots.net", ....